src/RunGui.cpp

```diff
@@ -111,9 +111,12 @@ namespace {
         QFile::remove(path);
     }
 
+#ifndef C_DEBUG
```

Comment:
how is this related to warning?

Reply:
In master, the warning is:

`RunGui.cpp:117:23: warning: unused function 'handleSignal'`

Because the function use is behind `#ifndef C_DEBUG`, so I put the function definition behind `#ifndef C_DEBUG` too, which leads to the variable not being used:

`RunGui.cpp:114:43: warning: unused variable 'signalsInitTime'`